BridgeSDKTests/SBBBridgeAPIIntegrationTestCase.m

```diff
@@ -49,23 +49,40 @@ - (void)setUp {
         
         // sensitive credentials are stored in a plist file that lives *outside* of the local git repo
         NSString *credentialsPlist = [[NSBundle bundleForClass:[self class]] pathForResource:@"BridgeAdminCredentials" ofType:@"plist"];
-        NSDictionary *credentials = [[NSDictionary alloc] initWithContentsOfFile:credentialsPlist][@"studies"];
-        NSDictionary *studyCredentials = credentials[[SBBBridgeInfo shared].studyIdentifier];
+        NSString *credentialsEmail = nil;
+        NSString *credentialsPassword = nil;
+        if (credentialsPlist) {
+            NSDictionary *credentials = [[NSDictionary alloc] initWithContentsOfFile:credentialsPlist][@"studies"];
+            NSDictionary *studyCredentials = credentials[[SBBBridgeInfo shared].studyIdentifier];
+            credentialsEmail = studyCredentials[@"email"];
+            credentialsPassword = studyCredentials[@"password"];
+        }
         
-        [gAdminAuthManager signInWithEmail:studyCredentials[@"email"] password:studyCredentials[@"password"] completion:^(NSURLSessionTask *task, id responseObject, NSError *error) {
-            if (error) {
-                NSLog(@"Error logging in to admin account:\n%@\nResponse: %@", error, responseObject);
-            } else {
-                NSLog(@"Logged in to admin account:\n%@", responseObject);
-            }
-            [expectAdminSignin fulfill];
-        }];
+        // ...if not found, check env vars
+        if (!credentialsEmail.length || !credentialsPassword.length) {
```

Comment:
I would prefer env vars to have precedence over file.  I think that would make testing easier since I can just pass in a var when i run a command instead of having to change a file.  for example:

SAGE_ADMIN_EMAIL="foo@sage.com" SAGE_ADMIN_PASSWORD="bar" fastlane mytest